### PR TITLE
Suggested Updates for Scaffold

### PR DESCRIPTION
**Pull request by tdm00**
Mr Riera,

Good afternoon.  Mike Henke introduced me to your plug-in during a "ColdFusion on Wheels" two day training class.  I really like how quickly it generates the MVC for me to get started and I see it as a very nice time saver, thanks for your contribution.

I've made a few changes that I feel maybe improvements for your consideration.  If you have questions, please let me know.

Thank you.

### CODE REVIEW

**Comment by tdm00** [comment]
Somehow the .gitignore was included, sorry about that, I'm still new with Git.  I just wanted to exclude the ANT build.xml file I was suing to ZIP the files and put them in the correct location on my system during testing.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+build.xml
```
Comment (tdm00) [inline .gitignore:2]: I added this for my system to not include the build.xml and it was included in the commit, sorry.

File: Scaffold.cfc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <cfcomponent output="false" mixin="controller" environment="design,development">
 
 	<cffunction name="init">
-		<cfset this.version = "1.0,1.0.1,1.0.2,1.1">
+		<cfset this.version = "1.0,1.0.1,1.0.2,1.1,1.1.4,1.1.5">
```
Comment (tdm00) [inline Scaffold.cfc:4]: This plug-in has been working for me on CFWheels 1.1.4 and 1.1.5 without any problems.  Therefore I updated the version number to remove the error message from displaying.

File: templates/cf8script/views/index.cfm [hunk 3]
```diff
@@ -1,12 +1,14 @@
 <h1>Listing [NamePluralLowercase]</h1>
 
+<cfoutput>#includePartial("showFlash")#</cfoutput>
+
+<cfoutput>
+	<p>#linkTo(text="New [NameSingularLowercase]", action="new")#</p>
```
Comment (tdm00) [inline templates/cf8script/views/index.cfm:6]: I realize the Scaffolding is just a quick way to generate something "that works" and not something you'd necessarily use "in production".  Sometimes the model already has a number of existing records and this link is at the very bottom.  I thought it made better sense to include it at the top of the list.

**Comment by raulriera** [comment]
I had a couple of pull request in there... Yours have been applied :) Can you make sure the current package works?

Thanks for helping out :)

---

Ral Riera
http://raulriera.com
@raulriera

**Comment by mhenke** [comment]
Raul, you are super fast :-)

**Comment by tdm00** [comment]
Raul,

I just downloaded the latest version of the files with the additions from Mike and myself and everything appears to be working correctly.  Looks like the CHANGELOG just needs to be updated, then the plug-in updated on the CFWheels plug-in directory.

Thanks!

**Comment by raulriera** [comment]
Excellent, 

Thanks guys.. will get right on that

**Comment by raulriera** [comment]
Updated, you guys rock :)

**Pull request merged**